requirements.txt

```diff
@@ -1,5 +1,5 @@
 psutil==2.1.3
-requests==2.5.0
+requests==2.7.0
```

Comment:
was this on purpose? if not, would prefer not to touch it because it might break something that we don't know about.